src/index.js

```diff
@@ -1,23 +1,30 @@
-import * as solparser from 'solidity-parser-sc'
+const solparser = require('@solidity-parser/parser');
 import { Graph } from  'graphlib'
 import * as dot from  'graphlib-dot'
 
-const SEND_NODE_NAME = 'UNTRUSTED'
-const SEND_NODE_STYLE = { shape: 'rectangle' }
+const DEPRECATED = ['send', 'transfer']
+
+const DEPRECATED_NODE_NAMES = {
+  send: 'DEPRECATED(send)',
+  transfer: 'DEPRECATED(transfer)',
+}
```

Comment:
These are derivable from `DEPRECATED`, so I don't think we need a separate readonly object.

Reply:
Agreed, i'm replacing it with a function
```suggestion
const deprecatedNodeName = name => `DEPRECATED(${name})`

```